src/cmd/services/m3dbnode/config/config.go

```diff
@@ -137,6 +137,9 @@ type DBConfiguration struct {
 
 	// Write new series asynchronously for fast ingestion of new ID bursts.
 	WriteNewSeriesAsync bool `yaml:"writeNewSeriesAsync"`
+
+	// Tracing configures opentracing. If not provided, tracing is disabled.
+	Tracing instrument.TracingConfiguration `yaml:"tracing"`
```

Comment:
i'd make this a pointer type to make it optional to specify

Reply:
This should be fine since we check if its a no-op in `server.go`:
```
	if _, ok := tracer.(opentracing.NoopTracer); ok {
		logger.Info("tracing disabled for m3dbnode; set `tracing.backend` to enable")
	}
```